ui/app/AppLayouts/stores/RootStore.qml

```diff
@@ -41,6 +50,20 @@ QtObject {
             name: "iconSource"
             expression: !!model.icon ? model.icon : tokenIcon(model.symbol)
         }
+        filters: [
+            AnyOf {
+                // We accept tokens from this community or general (empty community ID)
+                ValueFilter {
+                    roleName: "communityId"
+                    value: ""
+                }
+
+                ValueFilter {
+                    roleName: "communityId"
+                    value: root.communityKeyToImport
```

Comment:
FYI, this needs to be the public key

Reply:
yes, is `root.joinCommunity(d.publicKey, d.communityDetails);` and then we get `0x02c6a9e66f0153f9eaac2a25e2f8c343da5c31dcd6b2ed10f5c4350f4c67709248`

Reply:
@jrainville is this ok?